sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -325,7 +325,7 @@ private[joins] object UnsafeHashedRelation {
  * see http://java-performance.info/implementing-world-fastest-java-int-to-int-hash-map/
  */
 private[execution] final class LongToUnsafeRowMap(@transient val mm: TaskMemoryManager,
-  capacity: Int) extends MemoryConsumer(mm) with Externalizable {
+    capacity: Int) extends MemoryConsumer(mm) with Externalizable {
```

Comment:
I think this still does not follow [databricks/scala-style-guide#spacing-and-indentation](https://github.com/databricks/scala-style-guide#spacing-and-indentation).